Resources/doc/cookbook.md

```diff
@@ -162,3 +162,45 @@ $formatter = new Expr('function () {
 $ob->tooltip->formatter($formatter);
 $ob->series($series);
 ```
+
+## Map
+
+This is a simple recipe to create an FR map-chart demo at [highcharts.com/maps/demo/color-axis](http://www.highcharts.com/maps/demo/color-axis)
+
+In the controller file
+
+```php
+$map = new Highmap();
+$map->chart->renderTo('map');
+$map->chart->borderWidth(1);
+$map->title->text('FRANCE Population');
+$map->mapNavigation->enabled(true);
+$map->series(array(
+    array(
+        'data' => array(
+            array('value' => rand(1, 100), 'code' => 'Île-de-France'),
+            array('value' => rand(1, 100), 'code' => 'Bourgogne'),
+            array('value' => rand(1, 100), 'code' => 'Alsace'),
+            // ...
+        ),
+        'name' => 'FR',
+        'mapData' => new \Zend\Json\Expr("Highcharts.maps['countries/fr/fr-all']"),
```

Comment:
mapdata script, like the http://jsfiddle.net/gh/get/jquery/3.1.1/highslide-software/highcharts.com/tree/master/samples/maps/demo/color-axis/ example line 51